client/app/shared/services/notifications.service/index.d.ts

```diff
@@ -1,3 +1 @@
-declare namespace Foundation {
-  function Notification (message: string, level: string): void
-}
+declare var toastr;
```

Comment:
What is going on with this definition file. Where those the `toastr` come from and are we sure there is no way to require/import like normal js developers?

Reply:
The `toastr` and `jquery` are both part of the new UI package. jQuery is a basic dependency of sematnic-ui, and is defined globally by the UI package `toastr` is used for notifications as described in the [additional features](https://github.com/FirstLegoLeague/user-interface/blob/master/docs/additional-features.md) page of the [user-interface pacakge documentation](https://github.com/FirstLegoLeague/user-interface).